test/PrintAsCxx/lit.local.cfg

```diff
@@ -0,0 +1,12 @@
+# Make a local copy of the substitutions.
+config.substitutions = list(config.substitutions)
+
+config.substitutions.insert(0, ('%check-in-clang\+\+',
```

Comment:
So, this is just copied from `PrintAsObjC/lit.local.cfg`? Can we pull it into the parent directory or something so we don't have to duplicate it? (This is non-blocking.)

Reply:
I'd still kind of like to find a way to reorder these tests (but maybe it's not worth it). Even if not `Interop/Swift` maybe `Interop/{Forward,Reverse}/Cxx`? IDK, but it might be nice to group them better.

Reply:
Good point, I can take these two as action items to follow-up on. If we change around the directory structure we can create a shared lit local cfg for Interop that we can then reuse in PrintAsObjC/PrintAsCxx. Maybe we can have `Interop/SwiftToCxx` and `Interop/SwiftToObjC` directories where we can move the Print tests to?

Reply:
Actually, Let's do the following:
- Keep using `PrintAsCxx` to test the frontend option and high level flow into the C++ printer.
- Add `Interop/SwiftToCxx` to test semantics and do runtime testing of the emitted C++.
I will do this in a follow-up patch.